website/source/docs/guides/minikube.md

```diff
@@ -0,0 +1,198 @@
+
+```
+helm install -f helm-consul-values.yaml ./consul-helm
+```
+
+## Task 2: Deploy a Consul-aware Application to the Cluster
+
+Verify the installation by going back to the Kubernetes dashboard in your web browser. Find the list of services. Several include `consul` in the name and have the `app: consul` label.
+
+There are a few differences between running Kubernetes on a hosted cloud vs locally with minikube. You may find that any load balancer resources don't work as expected on a local cluster. But we can still view the Consul UI and other deployed resources.
+
+Run `minikube service list` to see your services. Find the one with `consul-ui` in the name.
+
+```
+minikube service list
+```
+
+Run `minikube service` with the `consul-ui` service name as the argument. It will open the service in your web browser.
+
+```
+minikube service original-hedgehog-consul-ui
+```
+
+You can now view the Consul web UI with a list of Consul's services, nodes, and other resources.
+
+###
+
+Now let's deploy our application. It's two services: a backend data service that returns a number (`counting` service) and a front-end `dashboard` that pulls from the `counting` service over HTTP and displays the number. The kubernetes part is a single line: `kubectl create -f 04-yaml-connect-envoy`. This is a directory with several YAML files, each defining one or more resources (pods, containers, etc).
+
+```
+kubectl create -f 04-yaml-connect-envoy
+```
+
+The output shows that they have been created. In reality, they may take a few seconds to spin up. Refresh the Kubernetes dashboard a few times and you'll see that the `counting` and `dashboard` services are running. You can also click a resource to view more data about it.
+
+###
+
+For the final step in this initial task, use the Kubernetes `port-forward` feature for the dashboard service running on port `9002`. We already know that the pod is named `dashboard` thanks to the metadata specified in the YAML we deployed.
+
+```
+kubectl port-forward dashboard 9002:9002
+```
+
+Visit http://localhost:9002 in your web browser. You'll see a running `dashboard` container in the kubernetes cluster that displays a number retrieved from the `counting` service using Consul service discovery and secured over the network by TLS via an Envoy proxy.
+
+###
+
+Let's take a peek at the code. Relevant to this Kubernetes deployment are two YAML files in the `04` directory. The `counting` service defines an annotation on lines 5 and 6 that instruct Consul to spin up a Consul Connect proxy for this service: `connect-inject`. The relevant port number is found in the `containerPort` section on line 12. This Pod registers a Consul service that will be available via a secure proxy.
+
+```yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: counting
+  annotations:
+    "consul.hashicorp.com/connect-inject": "true"
+spec:
+  containers:
+  - name: counting
+    image: topfunky/counting-service:0.0.1
```

Comment:
@topfunky should we push these images under HashiCorp docker image repository?

Reply:
Good point. I'm in the middle of getting access to the official HashiCorp DockerHub organization.

Reply:
Fixed in fba28d9